apps.json

```diff
@@ -9,11 +9,33 @@
                 "value": 2
             },
             "title": "Jupyter Notebook - General"
+        },
+        {
+            "app_name": "fas-jupyter-r",
+            "title": "Jupyter Notebook - R Kernel",
+            "docker_image": "jupyter/r-notebook",
```

Comment:
Fingers crossed this works as-is without requiring any additional modifications, either for FAS OnDemand or for the course.

Reply:
Yeah, the instructor only asked for tidyverse, which is already baked into the image. So, unless there is a surprise, we should be ok.